pique/functions.php

```diff
@@ -382,3 +382,24 @@ function pique_get_random_posts( $number = 1, $post_type = 'post' ) {
 	require get_template_directory() . '/inc/woocommerce.php';
 }
 
+/**
+ * Modifies Jetpack Testimonial Post Loop to include
+ * post ordering by menu_order
+ *
+ * New Order - Date - DESC, Menu Order - ASC
+ *
+ * @param array $query
+ * @return array $query
+ */
+function modify_jetpack_testimonial_archive_query( $query ) {
+	if ( $query->is_main_query() && is_post_type_archive( 'jetpack-testimonial' ) && ! is_admin() ) {
+		$query -> set(
+			'orderby',
+			array(
+				'menu_order' => 'ASC',
+			)
```

Comment:
This code will break the default testimonial order. Instead of

```
array(
  'menu_order' => 'ASC',
)
```

I'd use

```
array(
  'menu_order' => 'ASC',
  'date' => 'DESC',
)
```

to keep the default order.

Reply:
**Default order**

1. Create three testimonials: 
Jane Doe (Date: today | Time: 3:00 UTC | Order: 0)
John Doe (Date: today | Time: 2:00 UTC | Order: 0)
Jim Doe (Date: today | Time: 1:00 UTC | Order: 0)
2. Look up https://YOUR-TESTING-SITE/testimonial/

**Expected behaviour**

1. Jane Doe
2. John Doe
3. Jim Doe

**Current behaviour**

1. Jim Doe
2. John Doe
3. Jane Doe

---

**Custom order**

1. Adjust the order of the testimonials as follows
Jane Doe (Date: today | Time: 3:00 UTC | Order: 0)
John Doe (Date: today | Time: 2:00 UTC | Order: 1)
Jim Doe (Date: today | Time: 1:00 UTC | Order: 0)
2. Look up https://YOUR-TESTING-SITE/testimonial/

**Expected behaviour**

1. Jane Doe
2. Jim Doe
3. John Doe

**Current behaviour**

1. Jim Doe
2. Jane Doe
3. John Doe

Reply:
Thanks @nielslange. Fixed by adding date to array!